collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -315,35 +315,35 @@ private void processItemSets(File formMediaDir) {
     private boolean initializeForm(FormDef formDef, FormEntryController fec) {
         final InstanceInitializationFactory instanceInit = new InstanceInitializationFactory();
         boolean usedSavepoint = false;
+
         if (instancePath != null) {
-            File instance = new File(instancePath);
-            final File shadowInstance = SaveToDiskTask.getSavepointFile(instance.getName());
-            if (shadowInstance.exists() && (shadowInstance.lastModified()
-                    > instance.lastModified())) {
-                // the savepoint is newer than the saved value of the instance.
-                // use it.
+            File instanceXml = new File(instancePath);
+
+            // Use the savepoint file only if it's newer than the last manual save
+            final File savepointFile = SaveToDiskTask.getSavepointFile(instanceXml.getName());
+            if (savepointFile.exists() && (savepointFile.lastModified()
+                    > instanceXml.lastModified())) {
```

Comment:
Why another brackets here `(savepointFile.lastModified() > instanceXml.lastModified())` is it more clear for you?

Reply:
I just didn't remove them from the original code. It's true that they don't add anything. While I'm here I'll put the second test on a different line.